src/components/App/App.js

```diff
@@ -18,9 +18,13 @@ class App extends Component {
     this.setState({ currentCard: Number(event.target.id) })
   }
 
+  returnHome = (event) => {
```

Comment:
This is the most simple and beautiful logic, love how this works exactly how we want it to! Awesome work!